### DISCUSSION TITLE
Subsections expanded always on screen width < 1220px without need to click

### DISCUSSION THREAD

**Discussion by rofrol** [squidfunk/mkdocs-material#6089] [Q&A]
I have a request from one of the users of https://zighelp.org/ to use mdbook as it has saner navigation on mobile.

I would like to keep mkdocs-material as it offers more options and feels nicer.

I think this is responsible for navigation sections https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections,

but I don't know how to customize it, so that sections are always expanded also on small screens.

This is my config https://github.com/zighelp/zighelp/blob/master/mkdocs.yml

zighelp:

<img width="1058" alt="Screenshot 2023-09-26 at 14 24 46" src="https://github.com/squidfunk/mkdocs-material/assets/28787/2ed6a590-06f2-402b-8e1a-dae8d035bf4c">

mdbook:

<img width="1034" alt="Screenshot 2023-09-26 at 14 22 19" src="https://github.com/squidfunk/mkdocs-material/assets/28787/72108781-1b09-41e3-bcda-d64b76c8ba10">

**Comment by squidfunk** [comment]
Thanks for asking. IMHO, what is saner and what is not always depend on your point of view and the size of your site. Our mobile navigation is designed in a way that it scales without any additional effort, but I can understand that some users don't like that it is split into distinct panes. It's certainly possible to override navigation with custom CSS, making it look exactly like the one from mdbook, but it's likely some effort that you'd need to go through. You probably would also need to fork the theme, because otherwise you would have to override all properties for all elements, which would also be a maintenance nightmare once we make updates to the CSS.

That being said, we might add a new navigation design in the future, but there are currently no plans when this might be the case. We have other things that we need to fix first ☺️ Additionally, from our experience, mobile navigation is only used in 10% of cases – at least on our site, and we've had it for ages now. Note that it also works with JavaScript disabled.

**Reply by rofrol** [reply]
>to override navigation with custom CSS, making it look exactly like the one from mdbook, but it's likely some effort that you'd need to go through. You probably would also need to fork the theme, because otherwise you would have to override all properties for all elements, which would also be a maintenance nightmare once we make updates to the CSS.

That would be it. How can I start with that?

**Reply by squidfunk** [reply]
Fork the theme, set up the [development environment](https://squidfunk.github.io/mkdocs-material/customization/#theme-development), adjust `_nav.scss` – that's where all styles are defined, particularly this part, which defines the styles for mobile navigation, and [rebuild the theme](https://squidfunk.github.io/mkdocs-material/customization/#building-the-theme):

https://github.com/squidfunk/mkdocs-material/blob/929fc39dbe67dd5a2044636c81028270679d9213/src/templates/assets/stylesheets/main/components/_nav.scss#L182-L438
